docs/releaseNotes.yml

```diff
@@ -51,11 +59,25 @@ items:
           Users who rely on the specific ECMAScript Regex syntax will need to rewrite their regular
           expressions with RE2 syntax before upgrading to $productName$ 3.0.0.
 
-          Note that the <code>AMBASSADOR_ENVOY_API_VERSION</code> environment variable is now a
-          misnomer, as it no longer configures which xDS API version is used, but it still affects
-          what the default protocol used for a <code>TracingService</code> that points at Zipkin.
+          Note that the <code>AMBASSADOR_ENVOY_API_VERSION</code> environment variable is remove, 
+          it no longer configures which xDS API version is used.
+      - title: Support for protocol_version v2 is removed
+        type: change
+        body: >-
+          With the ugprade to Envoy 1.22, $productName$ no longer supports the V2 transport protocol.
+          The <code>AuthService</code>, <code>LogService</code> and the <code>RateLimitService</code> 
+          will only support the v3 protocol_version. If upgrading from a previous version you will want 
+          to explicitly set <code>protocol_version: "v3"</code> before upgrading to $productName$ 3.Y.
+          If protocol_version is not specified, the new default will be <code>v3</code>.
```

Comment:
Huh? We discussed that the default will continue to be `v2` and that `v2` will emit an error. That this is effectively now a required field.

Reply:
Based on our discussion, I have reverted the default back to v2 and updated release notes.